src/main/haskell/kore/src/Kore/Step/TranslateSMT.hs

```diff
@@ -0,0 +1,261 @@
+{-|
+Module      : Kore.Step.TranslateSMT
+Description : Evaluates conditions.
+Copyright   : (c) Runtime Verification, 2018
+License     : NCSA
+Maintainer  : thomas.tuegel@runtimeverification.com
+Stability   : experimental
+Portability : portable
+-}
+
+module Kore.Step.TranslateSMT
+    ( translatePredicate
+    , Translator
```

Comment:
This looks like a refactoring, so you can ignore this comment, but all exported stuff should be documented.